driver/driver_test.go

```diff
@@ -95,15 +95,20 @@ var _ = Describe("Contrail Network Driver", func() {
 		err := contrailDriver.StartServing()
 		Expect(err).ToNot(HaveOccurred())
 
-		d, err := sockets.DialPipe("//./pipe/"+common.DriverName, timeout)
+		conn, err := sockets.DialPipe("//./pipe/"+common.DriverName, timeout)
 		Expect(err).ToNot(HaveOccurred())
-		d.Close()
+		if conn != nil {
```

Comment:
Is there a possibility of both following statements being true at once?
* `conn` being `nil`
* `err` not occuring